client/src/pages/Signup.js

```diff
@@ -7,177 +7,227 @@ import {
     Row,
     Label,
     Input,
-    Button, 
+    Button,
     FormText
 } from 'reactstrap';
 
 class Signup extends React.Component {
-    constructor (props) {
+    constructor(props) {
         super(props);
 
         this.state = {
             // newUser: {
-                // this will hold an array of objects identifying the user
-                firstName: "",
-                lastName: "",
-                email: this.props.location.state.emailInput,
-                neighborhood: "",
-                password: "",
-                bio: "",
-                photo_url: ""
+            // this will hold an array of objects identifying the user
+            firstName: "",
+            lastName: "",
+            email: this.props.location.state.emailInput,
+            neighborhood: "",
+            residence: "",
+            entryway: "",
+            password: "",
+            bio: "",
+            skills: "",
+            photo_url: ""
             // }
         };
         console.log(this.state.firstName)
-    
+
         //bind the methods to "this"
         this.handleChange = this.handleChange.bind(this);
     }
-        //event handlers and functions to handle form submission
-        handleChange = event => {
-            const { name, value } = event.target
-            this.setState({
-                [name]: value,
-            })
-        }
+    //event handlers and functions to handle form submission
+    handleChange = event => {
+        const { name, value } = event.target
+        this.setState({
+            [name]: value,
+        })
+    }
 
-        handleSubmit = event => {
-            event.preventDefault();
-            console.log(`Form submitted: `, this.state);
-        }
+    handleSubmit = event => {
+        event.preventDefault();
+        console.log(`Form submitted: `, this.state);
+        // submit to backend to save might send you a token instead
+        // this should come from App.js
+        // this.props.logIn(<user obj from mongo>);
+    }
+
+    // componentWillMount () {
+    //     const { emailInput } = this.props.location.state
+    //     console.log(emailInput);
+    // }
 
-        // componentWillMount () {
-        //     const { emailInput } = this.props.location.state
-        //     console.log(emailInput);
-        // }
-    
     render() {
```

Comment:
This file is really starting to become big and unreadable. Is there any possiblity that you can make this into multiple components 🤔